src/reader/config.rs

```diff
@@ -62,7 +62,14 @@ pub struct ParserConfig {
     /// however, it is convenient to make the parser recognize additional entities which
     /// are also not available through the DTD definitions (especially given that at the moment
     /// DTD parsing is not supported).
-    pub extra_entities: HashMap<String, char>
+    pub extra_entities: HashMap<String, char>,
+
+    /// Tell the parser to ignore the end of file/stream
+    ///
+    /// By default this option is setted to false but if you want to deal with progressive inputs
+    /// like a stream over tcp, you can set it to true to make the parser ignore end of stream
+    /// error.
+    pub ignore_end_of_stream: bool
```

Comment:
I don't want to nitpick, but I prefer the documentation to be as clean as possible :)

> Whether or not the parser should ignore the end of stream. Default is false.
>
> By default the parser will either error out when it encounters a premature end of stream or complete normally if the end of stream was expected. If you want to continue reading from a stream whose input is supplied progressively, you can set this option to true. In this case the parser will allow you to invoke the `next()` method even if a supposed end of stream has happened.
>
> Note that support for this functionality is incomplete; for example, the parser will fail if the premature end of stream happens inside PCDATA. Therefore, use this option at your own risk.